web-api/src/persistence/dynamo/cases/getCaseByDocketNumber.ts

```diff
@@ -15,9 +15,11 @@ import { queryFull } from '../../dynamodbClientService';
 export const getCaseByDocketNumber = async ({
   applicationContext,
   docketNumber,
+  includeConsolidatedCases = true,
```

Comment:
Instead of adding this flag, can we allow the entity or interactor to strip out consolidatedCases?

Reply:
we surely could, it just felt like unnecessary dynamo queries when the first thing it does is remove it from memory.